df.advmode.xml

```diff
@@ -664,11 +664,11 @@
         <int32_t name='sleep_hours'/>
         <bool name='sleep_until_dawn'/>
         <int8_t name='unk_78'/>
-        <bool name='wait'/>
+        <bool name='sleep_not_wait' comment='Set when the player chooses to sleep instead of waiting'/>
```

Comment:
Could this be an int8_t enum instead? e.g. 
```xml
<enum name='wait_choice' base-type='int8_t'>
  <enum-item name='Wait'/>
  <enum-item name='Sleep'/>
</enum>
```
Might be clearer than having `not` in the name of a boolean.

Reply:
Yeah, definitely clearer. Something like `rest_mode`, perhaps?

Reply:
Sounds like a good name to me!